packages/browser/test/unit/transports/fetch.test.ts

```diff
@@ -451,6 +480,25 @@ describe('FetchTransport', () => {
         expect(eventRes.status).toBe(Status.Success);
         expect(fetch).toHaveBeenCalledTimes(2);
       });
+
+      it('should record dropped event', async () => {
+        // @ts-ignore private method
+        jest.spyOn(transport, '_isRateLimited').mockReturnValue(true);
+
+        const spy = jest.spyOn(transport, 'recordLostEvent');
+
+        try {
+          await transport.sendEvent(eventPayload);
+        } catch (_) {
+          expect(spy).toHaveBeenCalledWith(Outcome.RateLimitBackoff, 'event');
+        }
+
+        try {
+          await transport.sendEvent(transactionPayload);
+        } catch (_) {
+          expect(spy).toHaveBeenCalledWith(Outcome.RateLimitBackoff, 'transaction');
+        }
```

Comment:
Should this be a different unit test? I.e. one for the event, and another one for the transaction.

Reply:
I kept it in one test to make sure that `recordLostEvent` can be called with different inputs.